application/frontend/config/main.php

```diff
@@ -60,11 +60,12 @@
                 /*
                  * MFA routes
                  */
-                'GET    user/<employeeId:\w+>/mfa'    => 'mfa/list',
-                'POST   mfa'                          => 'mfa/create',
-                'PUT    mfa/<id:\d+>'                 => 'mfa/update',
-                'POST   mfa/<id:\d+>/verify'          => 'mfa/verify',
-                'DELETE mfa/<id:\d+>'                 => 'mfa/delete',
+                'GET    user/<employeeId:\w+>/mfa'              => 'mfa/list',
+                'POST   mfa'                                    => 'mfa/create',
+                'PUT    mfa/<id:\d+>'                           => 'mfa/update',
+                'POST   mfa/<id:\d+>/verify'                    => 'mfa/verify',
+                'DELETE mfa/<id:\d+>'                           => 'mfa/delete',
+                'DELETE mfa/<mfaId:\d+>/credential/<credId:.+>' => 'mfa/delete-credential',
```

Comment:
Would the `credId` also be numeric, like the `mfaId`? (e.g. `<credId:\d+>`)

Reply:
Not the way I've coded it.  At this point, `credId` is the same as the `key_handle_hash` that the mfa-api needs.

Should we consider making it related to the database record id?

Reply:
I think it would be better to use the database record ID, like we're doing for the `mfaId`.